.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,6 @@
+{
+	"image": "mcr.microsoft.com/devcontainers/universal:2",
+	"features": {
+		"ghcr.io/devcontainers/features/rust:1": {}
+	}
+}
```

Comment:
I think we should leave this out since this is a development environment-related config with little to do with the library itself. I think it'd be better to add `.devcontainer` to `.gitignore` instead.

Reply:
Haha whoops I didn't mean to include that, sorry.